docs-2.0/2.quick-start/2.deploy-nebula-graph-with-docker-compose.md

```diff
@@ -188,22 +222,28 @@ To update the nebula-console client, run the following command.
 docker pull vesoft/nebula-console:v2-nightly
 ```
 
-### How to upgrade Nebula Graph?
+### How to upgrade Nebula Graph service?
```

Comment:
services. Keep consistent with the following content. We are referring to the Graph Service, Meta Service, etc.